yetanotherco/aligned_layer
# feat (batcher): post batch to ethereum

Batcher now posts batches to ethereum. Reads configuration and deployment output from yaml files.

# To test:
1. Start anvil, aggregator, operator & batcher (make sure to have .env file filled with s3 credentials)
2. Run `make batcher_send_dummy_task` twice
3. Batcher should post 2 task batch to s3 and then to ethereum, then operator should verify both proofs & send response to aggregator
